site/src/components/Button/Button.tsx

```diff
@@ -51,9 +51,12 @@ export const Button: FC<ButtonProps> = forwardRef<
 	const Comp = asChild ? Slot : "button";
 	return (
 		<Comp
-			className={cn(buttonVariants({ variant, size, className }))}
 			ref={ref}
 			{...props}
+			className={cn(buttonVariants({ variant, size }), className)}
```

Comment:
We had a typo where we were accidentally passing the consumer class name into CVA, instead of passing it directly to `cn`